specs-go/config.go

```diff
@@ -30,7 +30,12 @@ type ContainerEdits struct {
 type DeviceNode struct {
 	HostPath      string   `json:"hostPath"`
 	ContainerPath string   `json:"containerPath"`
+	Type          string   `json:"type"`
+	Major         int64    `json:"major"`
+	Minor         int64    `json:"minor"`
 	Permissions   []string `json:"permissions,omitempty"`
+	UID           *uint32  `json:"uid,omitempty"`
+	GID           *uint32  `json:"umask,omitempty"`
```

Comment:
The json field name is incorrect. Should be `gid`. Also, should we drop the pointer specifier here? (not sure how json decoding works in this case).

Reply:
Are we sure we want to use the numeric uid/gid here and not the alphanumeric instead? There should be no problem to resolve such username/groupname is given otherwise I would consider that an error at startup.

Reply:
I was trying to be as close as possible to the [target structure](https://github.com/opencontainers/runtime-spec/blob/v1.0.2/specs-go/config.go#L377:L393).
Alphabetic uids can be distro-specific. Not sure we want that.